pages/upgrade/graylog-3.2.rst

```diff
@@ -9,6 +9,64 @@ Upgrading to Graylog 3.2.x
    :backlinks: top
 
 
+.. note:: Graylog 3.2.0 comes with a number of migrations that change parts of your existing data fundamentally. Please make sure to have a recent backup of your MongoDB that you can rollback to, before attempting to upgrade from an earlier version.
```

Comment:
I would make a warn and not only a note

Reply:
I updated the PR to make this a warning.